src/Features/CSharp/Portable/NavigationBar/CSharpNavigationBarItemService.cs

```diff
@@ -112,9 +112,12 @@ public CSharpNavigationBarItemService()
 
                     var symbolId = type.GetSymbolKey(cancellationToken);
                     items.Add(new RoslynNavigationBarItem.SymbolItem(
+                        type.Name,
                         text: type.ToDisplayString(s_typeFormat),
                         glyph: type.GetGlyph(),
+                        isObsolete: type.IsObsolete(),
                         spans: GetSpansInDocument(type, tree, cancellationToken),
+                        selectionSpan: GetSelectionSpan(type, tree),
```

Comment:
these values are added for LSP purposes.  note that spans/selectionSpan will be refactored again in my next pr which unifies span handling.  It will unify handling of both 'entire entity span' vs 'navigation span' as well as 'span to navigate to if in the starting document' vs 'span to navigate to in another document'.